### PR TITLE
add initial health module

### PR DESCRIPTION
**Pull request by tracy-french**
*Description of changes:*
Addition of health module.

Implementation completed following official NestJS recipe https://docs.nestjs.com/recipes/terminus.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by boweihan — DISMISSED** [review]
Cool :)

**Comment by tracfren** [comment]
@boweihan PR updated after resolving merge conflicts. Will need another approval. Thank you! :)

**Review by boweihan — DISMISSED** [review]

**Review by chejimmy — APPROVED** [review]
LGTM!

**Pull request merged**
